Assets/BossRoom/Scripts/Shared/Net/NetworkVariableGUID.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.IO;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace BossRoom
+{
+    [Serializable]
+    public class NetworkVariableGUID : NetworkVariableBase
```

Comment:
I'm guessing this branch isn't up to date with Val's latest changes?

Reply:
It is now!